wagtail/wagtailembeds/embeds.py

```diff
@@ -15,7 +15,11 @@ def get_embed(url, max_width=None, finder=None):
 
     # Get/Call finder
     if not finder:
-        finder = get_default_finder()
+        def finder(url, max_width=None):
+            for finder in get_finders():
+                if finder.accept(url):
+                    return finder.find_embed(url, max_width=max_width)
+
     embed_dict = finder(url, max_width)
 
```

Comment:
Entering a URL that's not recognised by any finder now fails here with `TypeError: 'NoneType' object does not support item assignment` (which suggests we're missing a test for this). This happens because the old code threw an `EmbedNotFoundException` from the finder (which was then caught by the UI code) - in the new code, the `finder` function above exits with `None`, having not found a suitable finder (and then breaks when we try to assign `embed_dict['width']`).

On the flip-side of this: if a finder *does* accept a URL but subsequently throws `EmbedNotFoundException`, we won't proceed to the next finder in the list - in particular, this means that any finder listed after `wagtail.wagtailembeds.finders.embedly` will never take effect.